sdk/communication/azure-communication-sms/src/test/java/com/azure/communication/sms/SmsClientTests.java

```diff
@@ -39,217 +29,145 @@ protected void beforeTest() {
 
     @ParameterizedTest
     @MethodSource("com.azure.core.test.TestBase#getHttpClients")
-    public void sendToIncorrectPhoneNumber(HttpClient httpClient) {
-        to = new ArrayList<String>();
-        to.add("+155512345678");
-        SmsClientBuilder builder = getSmsClientUsingConnectionString(httpClient);
-        client = setupSyncClient(builder, "sendToIncorrectPhoneNumber");
-        assertNotNull(client);
-        Iterable<SmsSendResult> response = client.send(SMS_SERVICE_PHONE_NUMBER, to, MESSAGE, null, Context.NONE);
-        assertNotNull(response);
-        for (SmsSendResult r : response) {
-            assertFalse(r.isSuccessful());
+    public void sendSmsToGroup(HttpClient httpClient) {
+        // Arrange
+        SmsClientBuilder builder = getSmsClient(httpClient);
+        client = setupSyncClient(builder, "sendSmsToGroupSync");
 
+        // Action & Assert
+        Iterable<SmsSendResult> sendResults = client.send(FROM_PHONE_NUMBER, Arrays.asList(TO_PHONE_NUMBER, TO_PHONE_NUMBER), MESSAGE);
+        for (SmsSendResult result : sendResults) {
+            assertHappyPath(result);
         }
     }
 
     @ParameterizedTest
     @MethodSource("com.azure.core.test.TestBase#getHttpClients")
-    public void sendFromFakeNumber(HttpClient httpClient) {
+    public void sendSmsToGroupWithOptions(HttpClient httpClient) {
+        // Arrange
+        SmsClientBuilder builder = getSmsClient(httpClient);
+        client = setupSyncClient(builder, "sendSmsToGroupWithOptionsSync");
         SmsSendOptions options = new SmsSendOptions();
         options.setDeliveryReportEnabled(true);
         options.setTag("New Tag");
-        // Arrange
-        SmsClientBuilder builder = getSmsClient(httpClient);
-        client = setupSyncClient(builder, "sendFromFakeNumber");
         // Action & Assert
-        try {
-            SmsSendResult response = client.send("+155512345678", SMS_SERVICE_PHONE_NUMBER, MESSAGE, options);
-        } catch (Exception exception) {
-            assertEquals(400, ((HttpResponseException) exception).getResponse().getStatusCode());
+        Response<Iterable<SmsSendResult>> sendResults = client.sendWithResponse(FROM_PHONE_NUMBER, Arrays.asList(TO_PHONE_NUMBER, TO_PHONE_NUMBER), MESSAGE, options, Context.NONE);
+        for (SmsSendResult result : sendResults.getValue()) {
+            assertHappyPath(result);
         }
     }
 
     @ParameterizedTest
     @MethodSource("com.azure.core.test.TestBase#getHttpClients")
-    public void sendFromUnauthorizedNumber(HttpClient httpClient) {
-        SmsSendOptions options = new SmsSendOptions();
-        options.setDeliveryReportEnabled(true);
-        options.setTag("New Tag");
+    public void sendSmsToSingleNumber(HttpClient httpClient) {
         // Arrange
         SmsClientBuilder builder = getSmsClient(httpClient);
-        client = setupSyncClient(builder, "sendFromUnauthorizedNumber");
+        client = setupSyncClient(builder, "sendSmsToSingleNumberSync");
```

Comment:
name should be without Sync.

Reply:
For all the Sync test cases, I've put "Sync" so that we can differentiate it from the async test cases. I didn't think it was necessary to suffix all the Async test cases with "Async" since it's a blit clunky.